installing/installing_aws/manually-creating-iam.adoc

```diff
@@ -10,29 +10,49 @@ toc::[]
 
 In environments where the cloud identity and access management (IAM) APIs are not reachable, or the administrator prefers not to store an administrator-level credential secret in the cluster `kube-system` namespace, you can put the Cloud Credential Operator (CCO) into manual mode before you install the cluster.
 
+//Alternatives to storing administrator-level secrets in the kube-system project
 include::modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc[leveloffset=+1]
 
-[role="_additional-resources"]
-.Additional resources
+For a detailed description of all available CCO credential modes and their supported platforms, see xref:../../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc#about-cloud-credential-operator[About the Cloud Credential Operator].
+
+//Supertask: Configuring a Google Cloud Platform cluster to use short-term credentials
+[id="installing-aws-with-short-term-creds_{context}"]
+=== Configuring an Amazon Web Services cluster to use short-term credentials
+
```

Comment:
Is it worth mentioning that configuring short-term credentials is done in two phases (not recommending `phases` but this is how I think of it.)

1. Prior to the installation, you complete the steps that you detail here.
2. As part of the installation process, you complete the configuration.

So maybe something like:

You can configure a cluster to use AWS short-term credentials. Before you begin the installation process:

- Configure the CCO utility.
- Use the CCO utility to create the AWS resources that the installation program requires to use this method of authentication.

Reply:
Thinking about this one... What do you think about making this step a "Preparing" (I'll work on the wording) intro, and then adding a Next steps to the module itself explainnig that the rest is done during install?

Reply:
@jeana-redhat Did you comment on this before our conversation in Slack?

Reply:
Per Slack conversation now: leaving this as-is for this PR, with a plan to completely remove this assembly in a future effort. Content can be covered in situ for install and post-install tasks, which should be a better UX.